jest.config.ts

```diff
@@ -225,6 +225,9 @@ const config: Config.InitialOptions = {
     // transform
     '^echarts/(.*)': '<rootDir>/tests/js/sentry-test/echartsMock.js',
     '^zrender/(.*)': '<rootDir>/tests/js/sentry-test/echartsMock.js',
+
+    '^preact$': require.resolve('preact'),
```

Comment:
Without this, everything (for whatever reason...) fails with ESM jest errors...

Reply:
guessing this is from the feedback integration... are our users going to run into this?

Reply:
yes they might do run into this. could we re-evaluate making it a dep and instead vendor it all in?

Reply:
I think vendoring would be better, it's weird that our app needs to know about sdk internals like this.

Reply:
Seems like it also needs to be updated in getsentry.

Reply:
Yeah, good point, I'll vendor this in! https://github.com/getsentry/sentry-javascript/pull/11845

Reply:
vendored in since beta.5, removed this config!